astropy/units/utils.py

```diff
@@ -144,13 +156,17 @@ def generate_prefixonly_unit_summary(namespace):
     docstring = io.StringIO()
 
     for unit_summary in _iter_unit_summary(faux_namespace):
-        docstring.write("""
+        docstring.write(
+            """
    * - Prefixes for ``{}``
      - {} prefixes
      - {}
      - {}
      - Only
-""".format(*unit_summary))
+""".format(
+                *unit_summary
```

Comment:
And here.

Reply:
Fixed both by making a template string. `.format` is fairly logical here.